manga_auto_translator/pipeline.py

```diff
@@ -1,10 +1,12 @@
 from typing import Sequence
 from manga_auto_translator.data_structure import Scan
+from manga_auto_translator.ocr import OcrStrategyFactory
 
 
 class TranslationPipeline:
-    def __init__(self, scans: Sequence[Scan]) -> None:
+    def __init__(self, scans: Sequence[Scan], pipeline_args: dict) -> None:
         self.scans = scans
+        self.ocr_strategy = OcrStrategyFactory(strategy=pipeline_args.get('ocr')).create()

```

Comment:
It's better to instanciate modules outside of the module scope and pass it through argument.
This way we can't properly test the module.

Reply:
The problem is that it takes forever to load the module, so I don't think we want to load it if we're not using it (so if we pass anything else than `--ocr manga-ocr`

Reply:
Oh nvm misread that I think, all good